resources/validations/src/ssp.sch

```diff
@@ -438,43 +408,1418 @@
+                        doc:attachment="§15 Attachment 2"
+                        id="has-user-guide"
+                        role="error"
+                        test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'user-guide']]">A
+                        FedRAMP OSCAL SSP must attach a User Guide.</sch:assert>
+            <sch:assert diagnostics="has-rules-of-behavior-diagnostic"
+                        doc:attachment="§15 Attachment 5"
+                        id="has-rules-of-behavior"
+                        role="error"
+                        test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'rules-of-behavior']]">A
+                        FedRAMP OSCAL SSP must attach Rules of Behavior.</sch:assert>
+            <sch:assert diagnostics="has-information-system-contingency-plan-diagnostic"
+                        doc:attachment="§15 Attachment 6"
+                        id="has-information-system-contingency-plan"
+                        role="error"
+                        test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'information-system-contingency-plan']]">
+            A FedRAMP OSCAL SSP must attach a Contingency Plan</sch:assert>
+            <sch:assert diagnostics="has-configuration-management-plan-diagnostic"
+                        doc:attachment="§15 Attachment 7"
+                        id="has-configuration-management-plan"
+                        role="error"
+                        test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'configuration-management-plan']]">
+                        A FedRAMP OSCAL SSP must attach a Configuration Management Plan.</sch:assert>
+            <sch:assert diagnostics="has-incident-response-plan-diagnostic"
+                        doc:attachment="§15 Attachment 8"
+                        id="has-incident-response-plan"
+                        role="error"
+                        test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'incident-response-plan']]">
+                        A FedRAMP OSCAL SSP must attach an Incident Response Plan.</sch:assert>
+            <sch:assert diagnostics="has-separation-of-duties-matrix-diagnostic"
+                        doc:attachment="§15 Attachment 11"
+                        id="has-separation-of-duties-matrix"
+                        role="error"
+                        test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'separation-of-duties-matrix']]">
+                        A FedRAMP OSCAL SSP must attach a Separation of Duties Matrix.</sch:assert>
+        </sch:rule>
+    </sch:pattern>
+    <sch:pattern>
+        <sch:title>Policy and Procedure attachments</sch:title>
```

Comment:
Can you add `[Section B Check X]` as appropriate per [the cross-referenced checklist in the README rendering of it](https://github.com/18F/fedramp-automation/blob/925e077a678cf90e1fab254be830908ccbe0dec9/resources/validations/README.md#section-b-documents-provided-check)?

Reply:
Yes. Will do after creating transform for Dan.